app/main/views/frameworks.py

```diff
@@ -136,6 +136,13 @@ def update_framework(framework_slug):
             enforce_required=True
         )
 
+    if json_framework.get('status') == 'expired' and framework.status != 'expired':
+        # framework_expires_at_utc is used in the buyer frontend in content that is displayed once the framework is
+        # expired. It's important that it's accurate. This ensures that the date is accurate with the framework is
+        # expired.
+        framework.framework_expires_at_utc = datetime.datetime.utcnow()
```

Comment:
We probably would want this recorded in the/an audit event as well

Reply:
Cool, have updated.